src/Conference/ConferenceContainer.jsx

```diff
@@ -52,7 +52,7 @@ const ConferenceContainer = () => {
         />
       </Route>
       <Route path="/:id">
-        <ConferenceDetails
+        <CookieConferenceDetails
```

Comment:
Don't think it should be prefixed with Cookie, Conference container doesn't care whether its involves cookies or not!